src/lib/pages/evm-tx-details/components/evm-tx-method/EvmTxCreateContract.tsx

```
@@ -75,22 +87,29 @@ export const EvmTxCreateContract = ({
           contracts.length > 0 ? (
             <Flex direction="column" gap={2}>
-              {contracts.map((contract) => (
-                <Flex key={contract} align="center" gap={1}>
-                  <CustomIcon
-                    boxSize={3}
-                    color="primary.main"
-                    name="contract-address"
-                  />
+              {contracts.map((contract) =>
+                isEvmVerifyInfosLoading ? (
+                  <Spinner boxSize={4} />
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)